### ISSUE TITLE
Release newui 

### ISSUE THREAD

**Issue by RacingTadpole** [TerriaJS/terriajs#1427]
Here are the tasks remaining to complete the `newui` branch:

Packages
- [x] Decide to use webpack and Babel.
- [x] Implement webpack and Babel.
- [x] Now that key changes have been merged into html-to-react, use an official release.

Feature Info Panel
- [x] Must update if the data has a time dimension.
- [x] Show message if >100 features.
- [x] Sanitises HTML.
- [x] Escapes HTML only as necessary (eg. currently shows & as `&amp;` in bird info).
- [x] Formats numbers.
- [x] Has working tests.
- [x] Shows html directly. Note:
  - Applying `<style>` tags in that html is hard - I propose we ignore them.
  - With the retirement of google maps engine, do we need this html to have a white background any more? I propose we show the html on our standard background.
- [x] **Show raw data** button.
- [x] **Download CSV** button.
- [x] Does not generate html-to-react warnings about `<br>`, `<img>` or `<table><span>`.
- [x] No feature info is shown for WFS layers, eg. on #test's data.gov.au link, "ABC Local Stations".
- [x] Support czml properties.
- [x] Support `properties._terria_columnAliases` from #1501.

Charts
- [x] MVP (as per #1278).
- [x] Run it past Phil - requested fixes for: similar line colors; tooltip padding; making preview y-axis standout if it hits the chart line.
- [x] Prevent all chart data from being cached for 2 weeks.
- [x] If raw data is passed in to `<chart>`, still allow it work with `<ChartExpandButton>`.

PreviewMap
- [x] Preview map stops updating sometimes, not sure what causes this to happen.
- [x] Fix #1436, which may involve a significant refactor of the terria object.
- [x] Apply the download-progress-bar to the preview panel.

Miscellaneous
- [x] csvs are requested 7 times (#1406). (Down to 3 - to go further depends on #1436.)
- [x] Drag and drop file UI sometimes get stuck. #1435 
- [x] When switching to full screen mode in 2d, Leaflet does not resize.
- [x] Leaflet does not resize when switching between mobile portrait and mobile landscape.
- [x] If the window height is short, the Terrarium welcome page on the preview window cannot be scrolled down at all, eg. to see the buttons.
- [x] Some information missing from catalog item info below preview. #1457
- [x] Viewing data in the preview can stop it appearing on the 2D map. #1469
- [x] Feedback form overlaps popup message window #1529

Tests
- [x] Improve shallow-render usage in tests.
- [ ] Add ReactView tests as needed (moved to #1686).

Analytics
- [ ] Print using browser print function (captures map, data and legend) (moved to #1687).
- [ ] See #1479 (Characteristics Selector/deselect)
- [ ] See #1480 (Adding the Criteria Selected to the Legend panel)

Browser Testing
- [x] Make it work on IE.
- [x] Specifically make `Object.assign` and `Map` work (or replace them).

UX
- [x] Implement alternative designs based on the user testing results.
- [ ] Move catalog search into a webworker (maybe) (moved to #1688).
- [x] Ensure all legends have titles as per prototype
- [x] Ensure "Share" is working as per prototype
- [ ] There's no way to access the group description/metadata on small-screen mobile (moved to #1658).
- [ ] There's no way to go back to the catalog once you preview a dataset on small screen mobile; you can only press Done to go all the way back to the map (moved to #1657).

General
- [ ] Do usual "pull request" review. (Will naturally occur when we raise the PR).
- [ ] Remove old unused ViewModels; rationalise those that are still required (moved to #1689).
- [x] Add back selectable text so we can Copy the text again.
- [ ] Make sure our various extension points are still available.  e.g. now viewing sections (moved to #1690).
- [x] Once we release, our React components and CSS need to be backward compatible.
- [x] Add a feedback mechanism (https://github.com/NICTA/nationalmap/issues/252).
- [x] Apply search improvements to the new UI (#1522)
- [x] Terrain height is continually polled on high DPI displays (#1523)
- [x] Group loading indicator is not shown once the group contains some items (#1524)
- [x] New UI doesn't use basemap-specified highlight color (#1528)
- [x] Side panel can't be scrolled #1532
- [x] Update the "About" page for NationalMap, http://nationalmap.gov.au/about.html

New issues
- [ ] Clicking on the chart panel doesn't bring it in front of the feature info panel any more (moved to #1691).
- [x] The map no longer resizes when opening/closing the chart panel.
- [x] If you "add data” when the chart panel is up, the add-data panel is half off the top of a small screen.
- [x] Remove mention of "Promise" from SearchBox.jsx (eslint error)
- [x] Remove mention of "confirm" from StandardUserInterface.jsx (eslint error)
- [x] When you drag a file onto the map, it only half slides the drag-n-drop screen onto the screen.

**Comment by kring** [comment]
> Preview map stops updating sometimes, not sure what causes this to happen.

Has anyone seen this since I refactored the preview?  I don't think I have.

**Comment by kring** [comment]
I don't see `confirm` in `StandardUserInterface.jsx` anymore, so I'm marking that one done.

**Comment by RacingTadpole** [comment]
Closing in favour of individual issues.
